index.html

```diff
@@ -736,27 +797,25 @@
           Emits an the event initialized with the event name specified by the <code>eventName</code> argument and data specified by the <code>payload</code> argument.
         </p>
         <p class="ednote">
-          This method in't actually needed to be exposed by the <a>ExposedThing</a> object since it is usually provided by the runtime.
+          This method isn't actually needed to be exposed by the <a>ExposedThing</a> object since it is usually provided by the runtime.
```

Comment:
Regarding the EDITOR’S NOTE of 6.1.18, I think emitEvent() is necessary for servient app.
For example, a script count event from a device, then when reaching a threshold issue an emitEvent().
Such kind of app could use the command. I might not catch the statement correctly though, I think it can be removed.